crates/bevy_color/Cargo.toml

```diff
@@ -16,7 +16,7 @@ bevy_reflect = { path = "../bevy_reflect", version = "0.14.0-dev", features = [
 bytemuck = "1"
 serde = "1.0"
 thiserror = "1.0"
-wgpu = { version = "0.19.1", default-features = false }
+wgpu = { version = "0.19.3", default-features = false }
```

Comment:
0.19.0, 0.19.1, 0.19.2 have the problematic dependency. The 0.19.3 is a real dependency.

I assume by "ignored by Cargo" you mean that Cargo prefers the latest version by default. Because with the 0.19.3 requirement Cargo will never build with anything lower.

Reply:
Cargo does use the minor version. This is specifying at least 0.19.3.

Reply:
Ah, right yep :)